locales/custom/en/common.json

```diff
@@ -0,0 +1,9 @@
+{
+  "entrinsik": "ACME",
+  "informer": "Reporting",
+  "entrinsik_informer": "ACME Reporting",
+  "informer4": "Legacy Reporting",
```

Comment:
the i4 key was removed from the core, might not want to mislead to its existence

Reply:
done